p/themes/Origine/origine.rtl.css

```
@@ -55,7 +55,8 @@ input:focus, select:focus, textarea:focus {
 	box-shadow: 0 2px 2px #ddf inset;
 }
 
-input:invalid, select:invalid {
+input:focus:invalid, select:focus:invalid,
```

This one too of course! ;-)
```suggestion
input:user-invalid, select:user-invalid,
input:focus:invalid, select:focus:invalid,
```

---

forgot it 🙈